conans/client/conan_api.py

```diff
@@ -3,11 +3,13 @@
 import sys
 from collections import OrderedDict
 from collections import namedtuple
-
 from io import StringIO
 
+from conans.model.manifest import discarded_file
```

Comment:
Use IDE optimize imports or any other tools, that will arrange the imports in order.